docs/import/manage_disaster.html

```diff
@@ -17,10 +17,11 @@
   <link href="../external_libs/jquery-ui-1.12.1.min.css" rel="stylesheet">
 
   <!-- Load debug Earth Engine JavaScript client library. -->
-  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): Switch back to compiled version in production. Compiled
```

Comment:
Still relevant TODO? When do you think this will be todone?

Reply:
See internal thread. They say this or next week?